src/components/Input/SelectList.js

```diff
@@ -90,6 +90,17 @@ export default class SelectList extends React.Component<Props, State> {
           renderItem={this.renderListItem}
           keyExtractor={({ name }) => name}
           keyboardShouldPersistTaps="handled"
+          initialNumToRender={10}
+          viewabilityConfig={{
+            minimumViewTime: 300,
```

Comment:
do we need this? on Android it causes content to disappear when scrolling fast and I don't think rendering a country name is so costly that we need such drastic optimisations

Reply:
yeah, makes sense, I'll make some adjustments